src/pages/home/report/ReportActionCompose.js

```diff
@@ -437,8 +592,33 @@ class ReportActionCompose extends React.Component {
      * @param {Object} e
      */
     triggerHotkeyActions(e) {
-        // Do not trigger actions for mobileWeb or native clients that have the keyboard open because for those devices, we want the return key to insert newlines rather than submit the form
-        if (!e || this.props.isSmallScreenWidth || this.props.isKeyboardShown) {
+        if (e.key === 'ArrowUp' && this.state.suggestedEmojis.length) {
```

Comment:
Can we please use the `ArrowKeyFocusManager` component for this logic?

Reply:
sure, updated